src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ParameterExpander.java

```diff
@@ -644,6 +646,22 @@ private String createBuildsStats(MemoryImprint memoryImprint, TaskListener liste
                             }
                         }
                     }
+
+                    if (build instanceof MatrixBuild && trigger.getIncludeMatrixResults()) {
```

Comment:
I was hoping to one day be able to yank out all direct dependencies on `MatrixBuild` into optional parts, so that users doesn't have to have the matrix plugin installed for this plugin to work.
But oh well, there are plenty of things in other places to fix for that anyway :)

Reply:
Thanks for taking a look! I'll add some tests.